docker/test

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -ex
+HERE=$(dirname $0)
+. $HERE/common
+
+NAME_SERVER=orderly_server
+
+function cleanup {
+  echo "Cleaning up"
+  docker kill $NAME_SERVER > /dev/null || true
+  rm -rf $PACKAGE_ROOT/orderly-demo
```

Comment:
This doesn't work here because the docker run creates some files

```
rm: cannot remove '/home/rob/projects/orderly.server/orderly-demo/runner/log': Permission denied
rm: cannot remove '/home/rob/projects/orderly.server/orderly-demo/runner/id': Permission denied
```

this is because it is bind mounted and creates these are root user. Not sure what the best way to deal with this is - I guessed you would have dealt something like this before? 

Tried copying instead but that doesn't work as we need the repo to exist when running the container, also tried changing ownership on the dir when it cloned but that doesn't seem ideal either.